apps/damap-frontend/src/themes/custom-theme.scss

```diff
@@ -169,6 +169,10 @@ html {
   --mat-icon-color: darkorange;
 }
 
+.language-color-btn {
+  color: var(--primary-lightest) !important;
```

Comment:
This should be rewritten to `--mdc-text-button-label-text-color: var(--primary-lightest);`. Important is not necessary, you should always overwrite the internal color variables of mat3 components. 

You could also rename it. When we need a white button in the future, we can then just reuse this class.